catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/FilterWriter.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.forms.filter;
+
+import java.io.StringWriter;
+import java.net.URL;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.validation.SchemaFactory;
+import net.opengis.filter.v_2_0.FilterType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xml.sax.SAXException;
+
+@SuppressWarnings("squid:S1075" /* Will parameterize only if necessary. */)
+public class FilterWriter {
+  private static final Logger LOGGER = LoggerFactory.getLogger(FilterWriter.class);
+
+  private static final String FILTER_XSD_RESOURCE_PATH = "/schemas/filter.xsd";
+
+  private static final String SCHEMA_LANGUAGE = "http://www.w3.org/2001/XMLSchema";
+
+  private final Marshaller marshaller;
+
+  /**
+   * Create a {@link FilterWriter}.
+   *
+   * <p>Accessing {@link javax.xml.XMLConstants#W3C_XML_SCHEMA_NS_URI} through the constants class
+   * causes an error: {@code javax.xml.XMLConstants cannot be found by catalog-ui-search}.
+   * Workaround is to use {@link #SCHEMA_LANGUAGE} constant instead.
+   *
+   * <p>See <a
+   * href="https://docs.oracle.com/javase/8/docs/api/javax/xml/validation/SchemaFactory.html">
+   * SchemaFactory</a> for more information.
+   *
+   * <p>See <a href="http://schemas.opengis.net/filter/2.0/">Filter 2.0</a> for original OGC schema
+   * documents.
+   *
+   * @param validationEnabled true if all XML writing done by this {@link FilterWriter} should also
+   *     be validated against the <a href="http://schemas.opengis.net/filter/2.0/filter.xsd">filter
+   *     schema</a>. False otherwise.
+   * @throws JAXBException if a problem occurs setting up and configuring JAXB.
+   */
+  public FilterWriter(boolean validationEnabled) throws JAXBException {
+    this.marshaller = JAXBContext.newInstance(FilterType.class).createMarshaller();
+    if (validationEnabled) {
+      LOGGER.info("Loading filter schemas");
```

Comment:
@Lambeaux But does it provide more benefit than overhead it occurs? In our [best practices page](https://codice.atlassian.net/wiki/spaces/DDF/pages/7045131/Log+Levels)
```
 - Issues that might be important, such as failing sources or an endpoint failing for an internal reason

 - Stack traces are acceptable at this level, but should be infrequent (do not log inside of a loop) and always include a descriptive message
```

There's many things we could alert people that are happening just to "let them know" its going good -- but then we'd be logging an outrageous amount of things that are going as planned. In the case of such a vast many logs it might be better to simply alert them when something **doesn't** go right since that will be the exception.

Reply:
Yeah, I think we should be following our logging guidelines

Reply:
Still going to disagree here. I believe I have followed the logging guidelines. The impact to bundle start time is worth logging at `INFO` because it is an issue that "might be important". If anything, a developer should be able to ascertain why the bundle isn't up yet, when compared against former preconceived notions. And again, this log will only occur once at system start. Not routinely as the system runs.

Reply:
Do you think that a developer could ascertain which bundles are not running out of all 500 bundles by noticing that one of them is missing? I don't believe so.